src/stores/AppStore.ts

```diff
@@ -774,12 +790,13 @@ export class AppStore {
         this.pendingChannelHistograms = new Map<string, CARTA.IRegionHistogramData>();
 
         this.frames = [];
-        this.catalogs = new Map();
         this.activeFrame = null;
         this.contourDataSource = null;
         this.syncFrameToContour = true;
         this.syncContourToFrame = true;
         this.initRequirements();
+        this.catalogs = new Map();
```

Comment:
better to explicitly specify the map type in the constructor call, to be consistent with the other maps